spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/token/TokenAuthenticationConverterTest.java

```diff
@@ -135,4 +138,16 @@ private static String getSidForAttributeValue(String attributeName, String attri
 		}
 
 	}
+
+
+	@Test
+	public void extractAuthoritiesWithScopesOAuth2Authenticaiton() {
+		Jwt jwt = new JwtGenerator().addScopes(scopeAdmin, scopeRead, scopeOther).getToken();
```

Comment:
I'm not satisfied with the test scope!!! If this is the only requirement of the uaa, this could be realized with the given TokenAuthenticationConverter.

Reply:
Ok it should test the type